mailpoet/prefixer/fix-symfony-polyfill.php

```diff
@@ -15,6 +15,9 @@
 $data = file_get_contents($file);
 $data = str_replace('\\Normalizer::', '\\MailPoetVendor\\Normalizer::', $data);
 $data = str_replace('\'Normalizer::', '\'\\MailPoetVendor\\Normalizer::', $data); // for use in strings like defined('...')
+// These unprefixed functions break WP 6.1 compatibility, we don't seem to use them, let's prefix them.
+$data = str_replace('function normalizer_is_normalized', 'function mailpoet_normalizer_is_normalized', $data);
+$data = str_replace('function normalizer_normalize', 'function mailpoet_normalizer_normalize', $data);
```

Comment:
I think we need to prefix the bootstrap (`vendor-prefixed/symfony/polyfill-intl-normalizer/bootstrap.php`) and not `vendor-prefixed/symfony/polyfill-intl-normalizer/Normalizer.php`.

I'm waiting for the release build to be completed so I can test it.

Reply:
You're right, I fixed it. Please refresh your local branch.